### ISSUE TITLE
Lost update was detected by jepsen append

### ISSUE THREAD

**Issue by zyguan** [tikv/tikv#10552]
### What version of TiKV are you using?

master

### What operating system and CPU are you using?

k8s

### Steps to reproduce

Run jepsen append workload.

### What did you expect?
The test passed.

### What did happened?
```
2021-07-10 14:36:12,631{GMT}    INFO    [jepsen test runner] jepsen.core: {:perf
 {:latency-graph {:valid? true},
  :rate-graph {:valid? true},
  :valid? true},
 :workload
 {:valid? false,
  :anomaly-types (:incompatible-order),
  :anomalies
  {:incompatible-order ({:key 3, :values [[4 1 2 5 3] [4 1 2 5 6]]})}},
 :valid? false}


Analysis invalid! (ﾉಥ益ಥ）ﾉ ┻━┻
```
[Here](http://fileserver.pingcap.net/download/pingcap/qa/tests/jepsen/logs/jepsen-append-nemesis-1625922677-1625927772.tar.gz) is the log.

**Comment by youjiali1995** [comment]
Fixed by https://github.com/tikv/tikv/pull/10551.

**Comment by zyguan** [comment]
Still failed after #10551 , the log can be found [here](http://fileserver.pingcap.net/download/pingcap/qa/tests/jepsen/logs/jepsen-async-commit-append1-1626206029-1626206426.tar.gz).

**Comment by zyguan** [comment]
> Still failed after #10551 , the log can be found [here](http://fileserver.pingcap.net/download/pingcap/qa/tests/jepsen/logs/jepsen-async-commit-append1-1626206029-1626206426.tar.gz).

It seems to be a new bug, open an another issue (#10570) for it.
